packages/devtools_app/test/standalone_ui/ide_shared/property_editor/property_editor_test.dart

```diff
@@ -99,6 +96,7 @@ void main() {
           mockEditorClient.getEditableArguments(
             textDocument: location.document,
             position: location.position,
+            screenId: 'propertyEditorSidebar',
```

Comment:
can we use the analytics constant here instead of the raw string?

Reply:
Yes, done!